jar-infer/scripts/android-jar.conf.template

```diff
@@ -1,19 +1,15 @@
+[base-config]
+nullaway-root = ${HOME}/NullAway
+sdk-version = 28
+nullaway-version = 0.5.3
+
 [android-paths]
-aosp-out-dir = /Volumes/AOSP8.1/out/target/common/obj/JAVA_LIBRARIES/framework_intermediates
-android-stubs-jar = ${HOME}/android-sdk/platforms/android-27/android.jar
+aosp-out-dir = /Volumes/AOSP/out/target/common/obj/JAVA_LIBRARIES/framework_intermediates
+android-stubs-jar = ${HOME}/android-sdk/platforms/android-%(sdk-version)s/android.jar
 
 [jar-infer-paths]
-jar-infer-path = ${HOME}/src/NullAway/jar-infer/jar-infer-cli/build/libs/jar-infer-cli-0.5.2-SNAPSHOT.jar
+jar-infer-path = %(nullaway-root)s/jar-infer/jar-infer-cli/build/libs/jar-infer-cli-%(nullaway-version)s.jar
 
 [android-model]
-wrk-dir = ${HOME}/src/NullAway/jar-infer/scripts/android-model
-android-model-jar = %(wrk-dir)s/android-jarinfer-model-0.5.2-SNAPSHOT.jar
+wrk-dir = %(nullaway-root)s/jar-infer/android-jarinfer-models-%(sdk-version)s
```

Comment:
Should this be %(nullaway-root)s/jar-infer/android-jarinfer-models-sdk%(sdk-version)s/src/main
Then the script will put it in the right place directly.

Reply:
Right 👍 (I'll test this script as soon as I manage to actually build AOSP)